bower_components/jquery/.bower.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
So in order to remove the bower_components files you've added and committed, you'll probably need to
```
git rm -r bower_components
git commit -m "Remove bower_components directory"
git push origin master
```
Afterwards you can get everything back with `bower install`